README.md

```diff
@@ -573,6 +573,79 @@ Don't worry about `adbkey` or `adbkey.pub` under `/.android`, not required.
+docker run -d -p 2222:22 -v $(pwd)/sdk:/opt/android-sdk -v /path/on/host/auth.json:/auth.json thyrlian/android-sdk-firebase-test-lab
+
+# check the running process and get a interactive shell
+docker exec -it `docker ps -aqf "ancestor=thyrlian/android-sdk-firebase-test-lab"` /bin/bash
+
+# authenticate and create any config that you want
+gcloud auth activate-service-account -q --key-file myservicefile.json
+
+# list available devices to run your tests
+gcloud firebase test android models list
+
+# you should get a matrix output similar to this one but bigger
+┌───────────────────┬────────────────────┬──────────────────────────────────────┬──────────┬─────────────┬─────────────────────────┬───────────────┐
+│      MODEL_ID     │        MAKE        │              MODEL_NAME              │   FORM   │  RESOLUTION │      OS_VERSION_IDS     │      TAGS     │
+├───────────────────┼────────────────────┼──────────────────────────────────────┼──────────┼─────────────┼─────────────────────────┼───────────────┤
+│ Nexus9            │ HTC                │ Nexus 9                              │ VIRTUAL  │ 2048 x 1536 │ 21,22,23,24,25          │               │
+│ NexusLowRes       │ Generic            │ Low-resolution MDPI phone            │ VIRTUAL  │  640 x 360  │ 23,24,25,26,27,28,29,30 │ beta=30       │
+│ OnePlus3T         │ OnePlus            │ OnePlus 3T                           │ PHYSICAL │ 1920 x 1080 │ 26                      │               │
+└───────────────────┴────────────────────┴──────────────────────────────────────┴──────────┴─────────────┴─────────────────────────┴───────────────┘
+
+# compile your apks both app and testing
+
+# to build the application apk
+./gradlew :<your_module>:assemble
+# apk will be generated at: <your_module>/build/outputs/apk/<build_type>
+
+# to build the test apk
+./gradlew :<your_module>:assembleAndroidTest
+# apk will be generated at: <your_module>/build/outputs/apk/androidTest/<build_type>
+
+# run your tests 
+UI_TEST_APK="--app=dir/debug.apk --test=dir/androidTest.apk"
+UI_TEST_TYPE="instrumentation"
+UI_TEST_DEVICES="--device model=MODEL_ID,version=OS_VERSION_IDS,locale=en,orientation=portrait" 
+UI_TEST_RESULT_DIR="build-result"
+UI_TEST_PROJECT="<your_project_id>"
+gcloud firebase test android run $UI_TEST_APK $UI_TEST_DEVICES --type=$UI_TEST_TYPE --results-dir=$UI_TEST_RESULTS_DIR --project=$UI_TEST_PROJECT
+
+# if you want to split your test suite into multiple parallel devices add the command --num-uniform-shards=int 
+# example: 20 test to run in 4 parallel devices (5 test per device) --num-uniform-shards=4
+
+# for more info about other configurations such as network-profile, locale, env variables etc ...  
+# go to https://cloud.google.com/sdk/gcloud/reference/beta/firebase/test/android/run
+```
+
+**Important**: To not lose your configurations copy ~/.config/gcloud to the host machine and
+pass it as external volume the next time you run the container.
```

Comment:
pass it as external volume the next time you run the container -> mount it next time